doc/userguide/rules/ip-reputation-rules.rst

```diff
@@ -17,20 +17,49 @@ The iprep directive matches on the IP reputation information for a host.
 
 side to check: <any|src|dst|both>
 
-category: the category short name
+``category``: the category short name
 
-operator: <, >, =
+``operator``: <, <=, >, >=, =
```

Comment:
There are more 

You could add a link to doc/userguide/rules/integer-keywords.rst